iree/testing/CMakeLists.txt

```diff
@@ -1,28 +1,41 @@
-################################################################################
-# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
-# iree/testing/BUILD                                                           #
-#                                                                              #
-# Use iree_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary   #
-# CMake-only content.                                                          #
-#                                                                              #
-# To disable autogeneration for this file entirely, delete this header.        #
-################################################################################
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Doesn't use bazel_to_cmake because of special logic for the benchmark library.
 
 iree_add_all_subdirs()
 
-iree_cc_library(
-  NAME
-    benchmark
-  HDRS
-    "benchmark.h"
-  SRCS
-    "benchmark_full.cc"
-  DEPS
-    benchmark
-    iree::base
-    iree::base::tracing
-  PUBLIC
-)
+if(${IREE_ENABLE_THREADING})
+  iree_cc_library(
+    NAME
+      benchmark
+    HDRS
+      "benchmark.h"
+    SRCS
+      "benchmark_full.cc"
+    DEPS
+      benchmark
+      iree::base
+      iree::base::tracing
+    PUBLIC
+  )
+else()
+  iree_cc_library(
+    NAME
+      benchmark
+    HDRS
+      "benchmark.h"
+    SRCS
+      "benchmark_nops.c"
```

Comment:
_nop (for consistency with iree/task/task_test_nop)

Reply:
Done.